docs/source/restore/restore-schema.rst

```diff
@@ -9,6 +9,9 @@ Restore schema
 .. include:: _common/restore-raft-schema-warn.rst
 
 In order to restore Scylla cluster schema use :ref:`sctool restore <sctool-restore>` with ``--restore-schema`` flag.
+Note that schema refers only to the data living in the ``system_schema`` keyspace (e.g. keyspace and tables definitions).
+All other data kept in keyspaces managed by ScyllaDB (e.g. authentication data living in ``system_auth`` keyspace)
+is restored as a part of :doc:`restore tables procedure <restore-tables>`.
```

Comment:
Maybe:
"Please note that the term 'schema' specifically refers to the data residing in the ``system_schema`` keyspace, such as keyspace and table definitions. All other data stored in keyspaces managed by ScyllaDB, such as authentication data in the ``system_auth`` keyspace, is restored as part of the :doc:`restore tables procedure <restore-tables>`."